vignettes/editors.Rmd

```diff
@@ -0,0 +1,156 @@
+---
+title: "Editor setup"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Editor setup}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## RStudio
+
+lintr lints are automatically displayed in the RStudio Markers pane, Rstudio versions (\> v0.99.206).
+
+![RStudio Example](http://i.imgur.com/PIKnpbn.png "Rstudio Example")
```

Comment:
is it best practice to save such png inside the vignettes directory? does the rendered HTML still require an internet connection to view it?

Reply:
In this case its about trade-offs — the advantage of using an external url is that you don't have to bundle the gif in the package, and hence distribute it individually to every one who uses the package. The cost is that you have to have an active internet connection to see it. The tradeoff seems worth it here since (a) the gif isn't that important to the user of the package and (b) I don't think there's an easy way to see the README offline from within R.

Reply:
I thought it was fine in the README, was asking bc this is in a vignette now